### PR TITLE
CIV-9455 Application Hearing Preferences and contact details

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-9455

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/views/features/generalApplication/hearing-contact-details.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{% extends "claim-details-tpl-dashboard.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "../../macro/contact-us-for-help.njk" import contactUsForHelp %}
+{% from "../../macro/heading-with-caption.njk" import headingWithCaption %}
+{% from "../../macro/citizen-telephone.njk" import citizenTelephone %}
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/views/features/generalApplication/hearing-contact-details.njk]: removed

File: src/main/views/features/generalApplication/hearing-arrangement.njk [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import {AgeHearingType} from "models/generalApplication/AgeHearingType";
+import {HearingTypeOptions} from "models/generalApplication/hearingType";
+import {CourtProposedPlanOptions} from "form/models/claimantResponse/courtProposedPlan";
+
+{% extends "claim-details-tpl-dashboard.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/views/features/generalApplication/hearing-arrangement.njk:4]: removed

File: src/main/views/features/generalApplication/hearing-contact-details.njk [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                label: {
+                    text: t('PAGES.GENERAL_APPLICATION.HEARING_CONTACT_DETAILS.PREFERRED_EMAIL_ADDRESS'),
+                    classes: "govuk-label--s"
+                },
+                errorMessage: {
+                    html: t(form.errorFor('emailAddress'))
+                } if form.hasErrors()
+            }) }}
+            <div class="govuk-button-group">
+              {{ govukButton({
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/views/features/generalApplication/hearing-contact-details.njk]: added addbutton macro

File: src/main/views/features/generalApplication/hearing-arrangement.njk [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+            {{ govukSelect({
+              id: "courtLocation",
+              name: "courtLocation",
+              items: courtItems,
+              label: {
+                text: t('PAGES.GENERAL_APPLICATION.APPLICATION_HEARING_ARRANGEMENTS.JUDGE_HEARING_PREFERRED_LOCATION'),
+                classes: "govuk-label--s"
+              }
+            }) }}
+            <div class="govuk-button-group">
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/views/features/generalApplication/hearing-arrangement.njk]: added addbutton macro

**Review by jeswanth-hmcts — APPROVED** [review]

**Review by kenneth-hmcts — DISMISSED** [review]

File: src/main/views/features/generalApplication/hearing-contact-details.njk [hunk 5]
```diff
@@ -0,0 +1,64 @@
+{% extends "claim-details-tpl-dashboard.njk" %}
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/views/features/generalApplication/hearing-contact-details.njk:1]: that should be refactor on the ticket: https://tools.hmcts.net/jira/browse/CIV-13893

**Review by laarmada — APPROVED** [review]
we will fix the design errors in: https://github.com/hmcts/civil-citizen-ui/pull/3775

**Pull request merged**